crates/astria-composer/src/searcher/collector.rs

```diff
@@ -73,16 +75,18 @@ impl Status {
 impl Collector {
     /// Initializes a new collector instance
     pub(super) async fn new(
-        chain_id: String,
+        chain_name: String,
         url: String,
         searcher_channel: Sender<Transaction>,
     ) -> eyre::Result<Self> {
         let client = EthClient::connect(&url)
             .await
             .wrap_err("failed connecting to eth")?;
         let (status, _) = watch::channel(Status::new());
+        let chain_id = ChainId::with_hashed_bytes(&chain_name);
```

Comment:
not sure how I feel about this, since the `chain_id` was supposed to just be some plaintext identifier of a chain, with a maximum length of 32. it's weird to have a chain_name and chain_id imo, can get confusing

Reply:
After thinking it through I agree. There is something off about demanding a fixed length.

At the same time I think a maximum length of 32 is too strict. Should we change this type to a `string` instead, and then internally hash it?